src/main/java/com/sap/oss/phosphor/fosstars/tool/format/OssRulesOfPlayRatingMarkdownFormatter.java

```diff
@@ -168,6 +194,25 @@ private String warningsFrom(ScoreValue scoreValue) {
 
     return String.format("## Warnings%n%n%s%n", content);
   }
+  
+  @Override
+  protected String formatRule(Value<?> value) {
+    return String.format("1.  **%s** %s **%s**",
+        identifierOf(value.feature()),
+        nameOf(value.feature()),
+        printValueAnswer(value));
+  }
+
+  private String printValueAnswer(Value<?> value) {
```

Comment:
Looks like this duplicates `formatRule()` a bit. We can try to get rid of dups later.